Tests/PlotTests/HTMLTests.swift

```diff
@@ -128,6 +128,55 @@ final class HTMLTests: XCTestCase {
         """)
     }
 
+    func testLinkHrefLangEnglish() {
+        let html = HTML(.head(.link(.rel(.alternate), .href("http://site/"), .hreflang(.english))))
+        assertEqualHTMLContent(html, """
+        <head><link rel="alternate" href="http://site/" hreflang="en"/></head>
+        """)
+    }
+
+    func testLinkHrefLangEnglishUS() {
```

Comment:
I don't think this test is needed anymore, since it tests the exact same code path.